shared/modules/provider-injection.js

```diff
@@ -61,9 +61,10 @@ function documentElementCheck() {
 /**
  * Checks if the current domain is blocked
  *
+ * @param currentUrl
  * @returns {boolean} {@code true} if the current domain is blocked
  */
-function blockedDomainCheck() {
+export function blockedDomainCheck(currentUrl) {
```

Comment:
Note: Realizing this now, if preferable I could remove the `currentUrl` argument and mock `window.location.href` in the tests